_includes/list-partners.html

```diff
@@ -0,0 +1,17 @@
+{% for agency in project.partner %}
+  {% capture agencyname %}{{ agency | slugify }}{% endcapture %}
+  {% capture partnerurl %}{% for partner in site.data.partners %}{% if partner[0] == agencyname %}{{ partner[1].url }}{% else %} {% endif %}{% endfor %}{% endcapture %}
+
+  {% if include.style == "list" %}
+    {% unless partnerurl == " " %}
+      <li><a href="{{ partnerurl }}">{{ agency }}</a></li>
+    {% else %}
+      <li>{{agency}}</li>
+    {% endunless %}
+  {% elsif include.style == "strong" %}
+    {% unless partnerurl == " " %}
+      <strong><a href="{{partnerur l}}">{{agency}}{% unless forloop.rindex == 1 %}, {% endunless%}</a></strong>
+    {% else %}
+      <strong>{{ agency }}</strong>
+  {% endif %}
+{% endfor %}
```

Comment:
Yeah, we've dealt with that before. I always forget about it but it's nuts how much whitespace gets left behind.